client/src/pages/search/index.js

```diff
@@ -20,40 +24,91 @@ export default function Search() {
   // papers : 1
   // both : 2
   // set default to videos ( 0 )
+
+  const [showMore, setShowMore] = useState(false);
+
+  const handleShowMore = () => {
+    setShowMore(!showMore);
+  };
+
   const SearchParams = useSearchParams();
   const Query = SearchParams ? SearchParams.get("query") : "";
   const Type = SearchParams ? SearchParams.get("type") : 0;
 
-  const { data, isLoading } = useSWR(
-    `http://127.0.0.1:5030/videos/search?query=${Query}`,
-    fetchResults
-  );
-
+  const data = [
```

Comment:
You are supposed to delete these things and return the code as it was after styling!